docker-compose.autoconftester.yaml

```diff
@@ -0,0 +1,130 @@
+version: '3'
+services:
+    autoconftester:
```

Comment:
autoconf tester needs to be built not here, but in travis workflow.

Reply:
yes it needs to follow the flow I have for the other tester in order to be able to copy the coverage reports